index.html

```diff
@@ -2203,12 +2197,12 @@ <h4>Proof Verification (Ed25519Signature2020)</h4>
 
             <ol class="algorithm">
               <li>
-Let `publicKeyBytes` be the result of retrieving the
+Let |publicKeyBytes| be the result of retrieving the
 public key bytes associated with the
-`options`.`verificationMethod` value as described in the
-Data Integrity [[VC-DATA-INTEGRITY]] specification,
-<a data-cite="vc-data-integrity#algorithms">
-Section 4: Retrieving Cryptographic Material</a>.
+|options|.|verificationMethod| value as described in the
+[[[controller-document]]] specification,
+<a data-cite="controller-document#retrieve-verification-method">
+Section: Retrieve Verification Method</a>.
```

Comment:
As above... `cid` spec?
```suggestion
|options|.|verificationMethod| value as described in the
<a data-cite="controller-document#retrieve-verification-method">Retrieve
Verification Method</a> section of the [[[controller-document]]] specification.
```